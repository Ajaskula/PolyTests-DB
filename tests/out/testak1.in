+2x^5 + 3x^4 + x^3 + 5x^2 - x - 4
*2x^5 - 3
*2
.
