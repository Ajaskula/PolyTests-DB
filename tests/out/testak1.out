2x^5 + 3x^4 + x^3 + 5x^2 - x - 4
4x^10 + 6x^9 + 2x^8 + 10x^7 - 2x^6 - 14x^5 - 9x^4 - 3x^3 - 15x^2 + 3x + 12
8x^10 + 12x^9 + 4x^8 + 20x^7 - 4x^6 - 28x^5 - 18x^4 - 6x^3 - 30x^2 + 6x + 24
